aim/db/model_base.py

```diff
@@ -99,6 +95,21 @@ class IsSynced(object):
 class AttributeMixin(object):
     """Mixin class for translating between resource and model."""
 
+    epoch = sa.Column(
+        sa.BigInteger().with_variant(sa.Integer(), 'sqlite'),
+        server_default='0', nullable=False)
```

Comment:
In the DB migration script, nullable is set to True. Is this discrepancy needed?

Reply:
Nope, will fix